scripts/apps/authoring-react/authoring-angular-integration.tsx

```diff
@@ -401,40 +515,158 @@ function getInlineToolbarActions(
 
     case ITEM_STATE.SPIKED:
         return {
-            readOnly: true,
-            actions: [], // un-spike
+            readOnly: false,
+            actions: [
+                {
+                    group: 'end',
+                    priority: 0.1,
+                    component: () => (
+                        <Button
+                            text={gettext('UNSPIKE')}
+                            style="filled"
+                            type="primary"
+                            onClick={() => {
+                                sdApi.article.doUnspike(item, item.task.desk, item.task.stage);
+                            }}
+                        />
+                    ),
+                    availableOffline: false,
+                },
+                closeIconButton,
+            ],
         };
 
     case ITEM_STATE.SCHEDULED:
         return {
-            readOnly: true,
-            actions: [], // un-schedule
+            readOnly: false,
+            actions: [{
```

Comment:
if it's scheduled, the only action is to de-schedule? No options to even close/minimize?